### DISCUSSION TITLE
Why won't we add whatsapp icon?

### DISCUSSION THREAD

**Discussion by habilm** [lucide-icons/lucide#2096] [Ideas]
As we're aware, WhatsApp is one of the most widely used instant messaging apps. It's often integrated as a sharing option on various websites and applications. By adding the WhatsApp icon to the Lucide library, we can greatly benefit developers who rely on the library, making it more comprehensive and user-friendly for everyone!

**Comment by karsa-mistmere** [comment]
#670

**Comment by Vi-fly** [comment]
one year update , nothing is down yet
